translations/messages+intl-icu.de.yml

```diff
@@ -1277,6 +1277,10 @@ error.statement.submit.nowriteaccess: "In dem Verfahren können derzeit keine St
 error.statement.submit.phase: "In der aktuellen Phase des Verfahrens können keine Stellungnahmen eingereicht werden."
 error.statement.update: "Beim Aktualisieren der Stellungnahme ist ein Fehler aufgetreten."
 error.statements.marked.submitted: "Die Stellungnahmen konnten nicht eingereicht werden!"
+error.statements.zip.export.generic.xlsx: "Die XLSX-Datei konnte dem ZIP-Archiv nicht hinzugefügt werden."
+error.statements.zip.export.generic.attachment: "Beim Hinzufügen eines Anhangs ist ein unbekannter Fehler aufgetreten."
+error.statments.zip.export.filenotfoundorreadable: "Der Anhang mit dem Hash: {hash} konnte dem ZIP-Archiv nicht hinzugefügt werden."
+error.statments.zip.export.hash.invalid: "Ein Anhang konnte dem ZIP-Archiv nicht hinzugefügt werden. Hash ist invalide."
```

Comment:
'invalide' in German is not the same as 'invalid' in English. I'd use 'ungültig' here, but again I'm wondering if this information is helpful for the user (if it's a message displayed to the user)

Reply:
These messages are included in a text file within the created zip file. adjusted in [8afdf43](https://github.com/demos-europe/demosplan-core/pull/2337/commits/8afdf439a2803ed7db991a32fd37abac44fb2329)